rules.neon

```diff
@@ -52,6 +52,10 @@ rules:
 	- PHPStan\Rules\VariableVariables\VariableStaticPropertyFetchRule
 	- PHPStan\Rules\VariableVariables\VariableVariablesRule
 
+conditionalTags:
+	PHPStan\Rules\DisallowedConstructs\DisallowedLooseComparisonRule:
+		phpstan.rules.rule: %featureToggles.bleedingEdge%
```

Comment:
@ondrejmirtes We are using bleeding edge to test `@readonly` and stuff in phpstan, but this rule would create thousands of errors for us in legac apps.

wouldn‘t it make sense to have a additional feature flag, so one can enable bleeding edge but don‘t enable this rule?

Reply:
Either we want to have this enabled for everyone when PHPStan 2.0 + phpstan-strict-rules 2.0 are released, or we don't.

In the first case (enable for everyone), the additional feature toggle doesn't make sense. Bleeding edge is a preview of what's to come in the next major version.

I made a Twitter poll about this: https://twitter.com/OndrejMirtes/status/1537418180888514560

Reply:
BTW - instead of setting a feature toggle to false or whatever, you can always write a regex to ignore these problems.